code/modules/hydroponics/seeds.dm

```diff
@@ -435,6 +435,14 @@
 		else
 			qdel(T)
 
+/obj/item/seeds/proc/add_random_glow()
+	var/random_trait = pick(subtypesof(/datum/plant_gene/trait/glow))
```

Comment:
This will result in shadow emission. just put `-/datum/plant_gene/trait/glow/shadow`

Reply:
also, I recommend using static.

Reply:
Static what, static list of glow traits?

Reply:
Also I'm fine with it including Shadow Emission btw

Reply:
yeah, `subtypesof(/datum/plant_gene/trait/glow)` will be always newly formed whenever the proc happens.